src/api/config/brakeman.ignore

```diff
@@ -249,18 +230,18 @@
     },
     {
       "warning_type": "Unmaintained Dependency",
-      "warning_code": 122,
-      "fingerprint": "98b26f60d776fd41ee6f088c833725145be9aac2d7c5b33780241c273622db42",
+      "warning_code": 120,
+      "fingerprint": "d84924377155b41e094acae7404ec2e521629d86f97b0ff628e3d1b263f8101c",
       "check_name": "EOLRails",
-      "message": "Support for Rails 7.0.8.1 ends on 2025-04-01",
+      "message": "Support for Rails 7.0.8.7 ended on 2025-04-01",
```

Comment:
You don't "update" this EOL warning, you ignore it.

Reply:
`Support for Rails 7.0.8.7 ended on 2025-04-01` yet we are still on it. So brakeman complains rightfully.